app/src/components/header.js

```diff
@@ -34,7 +31,7 @@ class Header extends React.Component {
         ) : null}
         <Grid columns={10} gap='15px' alignContent='center'>
           <Cell width={9} className='logo'>
-            <img src={logo} alt='logo' />
+            <img src='/static/images/logo.svg' alt='logo' />
```

Comment:
Wouldn't it be better to keep this outside so that it's checked when "compiling"?

Reply:
@syk0saje I wanted to reuse the static file in django. Wanted to do this by using the manifest file but still need to read up more on it.  Only getting a small subset of the generated files: ```{
  "index.css": "index.0e6922093e490fe116b5f05deecfd895.css",
  "index.js": "index.138b1e01763d27dfe0be.js",
  "runtime.js": "runtime.a9edc78d5cd6127c9851.js",
  "tablemap.svg": "tablemap.a58ce6e6684c2395b58a0495019e5cdc.svg"
}```